### ISSUE TITLE
Unable to use gpu acceleration as docker can't identify nvidia gpu driver

### ISSUE THREAD

**Issue by arkhamHack** [OpenDroneMap/NodeODM#187]
### What is the problem?

For running nodeodm:gpu I need nvidia gpu acceleration but docker is unable to find the nvidia gpu driver.
![image](https://user-images.githubusercontent.com/72064090/189491003-20d4c343-01c4-4660-a239-73af29f10fc1.png)

### What should be the expected behavior?
Ans: Nodeodm should be able to harness nvidia gpu fpr gpu acceleration.

### How can we reproduce this? (What steps did you do to trigger the problem? Be detailed)
Ans:
I downloaded the required nvidia driver and ran the command: 
docker run -p 3000:3000 --gpus all opendronemap/nodeodm:gpu

This uses the inbuilt intel gpu and not the nvidia gpu.
I have also installed wsl2 for setting up docker wsl2 backend.

![image](https://user-images.githubusercontent.com/72064090/189491139-fa989396-11f8-4685-ba82-b5c3526460e6.png)

**Comment by cbjornram** [comment]
Hey, saw this while passing by.
It looks like a non-ODM issue and more of a NVIDIA/Docker issue, since the first image you have attached shows that the Docker NVIDIA test fails. The `docker run --rm --gpus all nvidia/cuda:10.0-base nvidia-smi` test is completely independent from NodeODM and ODM.

Have you followed the NVIDIA Docker installation guide over at https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#docker?

On my system (Ubuntu 20.04) I first had to install NVIDIA drivers (using `sudo apt install nvidia-driver-460-server`), then I installed support for it in Docker with `sudo apt-get install -y nvidia-docker2`, before restarting and testing the `docker run --rm --gpus all nvidia/cuda:10.0-base nvidia-smi` command. If this fails, something with the NVIDIA installation itself has failed and must be solved before trying to run NodeODM with GPU.
